tests/integration-tests/src/commonTest/kotlin/test/WatcherTest.kt

```diff
@@ -295,7 +296,7 @@ class WatcherTest {
     val channel = Channel<EpisodeHeroNameQuery.Data?>(capacity = Channel.UNLIMITED)
     val job = launch {
 
-      apolloClient.query(EpisodeHeroNameQuery(Episode.EMPIRE))
+      apolloClient.query(EpisodeHeroNameQuery(Optional.Present(Episode.EMPIRE)))
```

Comment:
Use `generateOptionalVariables.set(false)` in `integration-tests` to keep the tests leaner and unchanged?

Reply:
I was hesitating about that and thought it would be interesting to see the impact of the change in our tests 😅 But now that we saw it, I can revert yes.